corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -317,6 +317,67 @@ def test_case_tile_suite(self):
         self._test_generic_suite("app_case_tiles", "suite-case-tiles")
 
 
+class ModuleAsChildTest(SuiteTest):
```

Comment:
Shouldn't extend SuiteTest otherwise all the tests in the base class will get run as well.

If you want to use a base class then create one that they both can use.